File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/ride/RideSummaryResponse.java
package com.lattis.ellipse.data.network.model.response.ride;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

/**
 * Created by ssd3 on 4/11/17.
 */

public class RideSummaryResponse extends AbstractApiResponse {

    @SerializedName("payload")
    RideSummaryPayloadResponse rideSummaryPayloadResponse;

    public RideSummaryDataResponse getRideSummaryResponse() {
        return rideSummaryPayloadResponse.getRideSummaryDataResponse();
    }

    @Override
    public String toString() {
        return "RideSummaryResponse{" +
                "RideSummaryResponse=" + rideSummaryPayloadResponse +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/ride/RideSummaryResponse.kt
package com.lattis.data.entity.response.ride

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse


class RideSummaryResponse : AbstractApiResponse() {

    @SerializedName("payload")
    internal var rideSummaryPayloadResponse: RideSummaryPayloadResponse? = null

    val rideSummaryResponse: RideSummaryDataResponse?
        get() = rideSummaryPayloadResponse!!.rideSummaryDataResponse

    override fun toString(): String {
        return "RideSummaryResponse{" +
                "RideSummaryResponse=" + rideSummaryPayloadResponse +
                '}'.toString()
    }
}
